### PR TITLE
Add ability to switch DAO

### LINKED ISSUE #31: Support Multiple DAOs
#### Description

- Add the dao address to the router
- Add a dropdown with all of the daos that allows a user to switch between daos.

### PR DESCRIPTION
**Pull request by alexkeating**
#### Description

- Add ability to switch daos

Closes #31

### CODE REVIEW

**Comment by alexkeating** [comment]
Current dependencies on/for this PR:
* main
  * **PR #106** <a href="https://app.graphite.dev/github/pr/ScopeLift/l2-flexible-voting-frontend/106" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/ScopeLift/l2-flexible-voting-frontend/106?utm_source=stack-comment).

**Review by garyghayrat — APPROVED** [review]
lgtm!

File: config.ts [hunk 1]
```diff
@@ -35,23 +37,30 @@ export const config = {
...
     l1: {
       chain: goerli,
       wormholeChainId: 2,
+      deployBlock: 9895307n,
       tokenAddress: '0x449FdC2D5038E0447801025e8Eb47DD8b68a7B90' as `0x${string}`,
       tokenDecimals: 18,
       erc20Bridge: '0x46A5D1C7c2eF4d276562DB6f504b25bD88CfeeD2' as `0x${string}`,
       governor: '0xA4c74075770582ad5873E3d9e288f7877BD90c85' as `0x${string}`,
       metadataBridge: '0x49F8948E4455Eb1d8D3964a9DC17f831C12B0893' as `0x${string}`,
+      tallyGovernorDomain: '',
```
[the remark this answers is not in the document]
Comment (wildmolasses) [reply config.ts]: ah, you mean the governor can only be listed after the contract is verified?

File: config.ts [hunk 2]
```diff
@@ -9,8 +9,10 @@ export enum DaoId {
 
 export const config = {
   [DaoId.Example]: {
-    name: 'Example Governor',
+    name: 'Example',
+    id: DaoId.Example,
```
Comment (wildmolasses) [inline config.ts:13]: (arbitrary place to leave comment)

given the new routing, we might want to prefer urls with an identifying string in them (aka */pooltogether/bridge*) which is easily done via string enum for DaoId type:

i.e. 
```
enum DaoId {
  Example = "pooltogether",
  ExampleComp = "example-comp"
}
```

if you agree, happy to call it follow up and not block merge

File: components/DaoMenu.tsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  value: T;
+  label: string;
+  logo: string;
+};
+
+type Props<T> = {
+  options: Option<T>[];
+};
+
+const DaoMenu = <T,>(props: Props<T>) => {
```
Comment (wildmolasses) [inline components/DaoMenu.tsx:21]: Curious why we're using generics here

File: util/constants.ts [hunk 4]
```diff
@@ -1 +1,2 @@
 export const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000';
```
Comment (wildmolasses) [inline util/constants.ts:1]: (arbitrary place to leave a comment)

netlify.toml should also be updated:

```toml
[[redirects]]
  from = "/"
  to = "/pool-together/bridge"
  status = 302
```

**Review by wildmolasses — APPROVED** [review]
awesome work! a couple minor comments, curious what you think

File: config.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (alexkeating) [reply config.ts]: I think so, I ended up verifying the contracts this morning and creating the DAOs on Tally.

File: config.ts (hunk 2 above)
Comment (alexkeating) [reply config.ts:13]: My one worry with that scheme is we can run into future problems if names aren't unique. Although, I don't see that being an issue anytime soon. We could do `/[id]/[name]/bridge`?

File: util/constants.ts (hunk 4 above)
Comment (alexkeating) [reply util/constants.ts:1]: Good catch!

File: components/DaoMenu.tsx (hunk 3 above)
Comment (alexkeating) [reply components/DaoMenu.tsx:21]: I used it mainly to give us flexibility if we decided to change types for the value. Right now, we are using numbers, but like you suggested we might want to switch to strings.

File: config.ts (hunk 2 above)
Comment (alexkeating) [reply config.ts:13]: Happy to flesh out more in an issue as well https://github.com/ScopeLift/l2-flexible-voting-frontend/issues/115

**Pull request merged**